openmc/model/model.py

```diff
@@ -1,11 +1,20 @@
 from collections.abc import Iterable
+import os
 from pathlib import Path
+from numbers import Integral
 import time
+import warnings
+import subprocess
 
 import h5py
+import numpy as np
 
 import openmc
-from openmc.checkvalue import check_type, check_value
+from openmc.checkvalue import check_type, check_value, check_iterable_type, \
+    check_length
+import openmc.deplete as dep
```

Comment:
The reason I moved it globally is because of the following:

1. I'm probably too used to it now, but I was thinking that having the shared library around is no longer a big hurdle to overcome, especially since so much functionality requires it now.

2. I have an MPI barrier when writing the XML files. I realize now that the better approach could just be to set the communicator at ``Model.__init__``

Reply:
This is now done